tests/conftest.py

```diff
@@ -0,0 +1,86 @@
+import logging
```

Comment:
While implementing the load testing, I used the forlder `./packages/tests/k6`shouldn't be useful to have both tests artifacts under the same folder.
- Option 1: `./packages/test/k6` and `./packages/test/e2e`
- Option 2: `./tests/k6`and `./tests/e2e`

Reply:
I'm leaning towards option 2, good catch.

Reply:
Solved by https://github.com/hoprnet/hoprnet/pull/4771